src/snippets/curl/contacts_api/_patch_contacts.mdx

```diff
@@ -1,6 +1,6 @@
 ```bash
   curl -X PATCH "https://api.callbell.eu/v1/contacts/414a6d692bd645ed803f2e7ce360d4c8" \
       -H "Authorization: Bearer test_gshuPaZoeEG6ovbc8M79w0QyM" \
-      -H "Content-Type: application/json"
+      -H "Content-Type: application/json" \
```

Comment:
Note: not sure on how was supposed to work but this is not valid cURL syntax 🤔 I had to add the missing backslash to make the script work.

Reply:
Oh nice, not sure why it was missing, thanks for fixing it 🙏🏼